task_10/js/config.js

```diff
@@ -0,0 +1,251 @@
+const key = "key=91e53c3974b54ac9871fe08adfd31dd9";
```

Comment:
Not good. `key=` is not part of key, is it?

Reply:
Fixed.